qlib/mod.rs

```diff
@@ -126,26 +126,47 @@ pub fn InitSingleton() {
     }
 }
 
-pub const HYPERCALL_PANIC: u16 = 2;
-pub const HYPERCALL_OOM: u16 = 4;
-pub const HYPERCALL_MSG: u16 = 5;
-pub const HYPERCALL_U64: u16 = 6;
-pub const HYPERCALL_PRINT: u16 = 8;
-pub const HYPERCALL_EXIT: u16 = 32;
-pub const HYPERCALL_GETTIME: u16 = 11;
-pub const HYPERCALL_QCALL: u16 = 12;
-pub const HYPERCALL_HLT: u16 = 13;
-pub const HYPERCALL_URING_WAKE: u16 = 14;
-pub const HYPERCALL_HCALL: u16 = 15;
-pub const HYPERCALL_IOWAIT: u16 = 16;
-pub const HYPERCALL_WAKEUP_VCPU: u16 = 17;
-pub const HYPERCALL_EXIT_VM: u16 = 18;
-pub const HYPERCALL_VCPU_FREQ: u16 = 19;
-pub const HYPERCALL_VCPU_YIELD: u16 = 20;
-pub const HYPERCALL_VCPU_DEBUG: u16 = 21;
-pub const HYPERCALL_VCPU_PRINT: u16 = 22;
-pub const HYPERCALL_VCPU_WAIT: u16 = 23;
-pub const HYPERCALL_RELEASE_VCPU: u16 = 24;
+//pub const HYPERCALL_PANIC: u16 = 2;
+//pub const HYPERCALL_OOM: u16 = 4;
+//pub const HYPERCALL_MSG: u16 = 5;
+//pub const HYPERCALL_U64: u16 = 6;
+//pub const HYPERCALL_PRINT: u16 = 8;
+//pub const HYPERCALL_EXIT: u16 = 9;
+//pub const HYPERCALL_GETTIME: u16 = 11;
+//pub const HYPERCALL_QCALL: u16 = 12;
+//pub const HYPERCALL_HLT: u16 = 13;
+//pub const HYPERCALL_URING_WAKE: u16 = 14;
+//pub const HYPERCALL_HCALL: u16 = 15;
+//pub const HYPERCALL_IOWAIT: u16 = 16;
+//pub const HYPERCALL_WAKEUP_VCPU: u16 = 17;
+//pub const HYPERCALL_EXIT_VM: u16 = 18;
+//pub const HYPERCALL_VCPU_FREQ: u16 = 19;
+//pub const HYPERCALL_VCPU_YIELD: u16 = 20;
+//pub const HYPERCALL_VCPU_DEBUG: u16 = 21;
+//pub const HYPERCALL_VCPU_PRINT: u16 = 22;
+//pub const HYPERCALL_VCPU_WAIT: u16 = 23;
+//pub const HYPERCALL_RELEASE_VCPU: u16 = 24;
```

Comment:
If there is no further need for this lines, it will be better to remove them at all.